_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -19,32 +19,46 @@ practitioners with interests in the methodological, algorithmic, and software as
 [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
 we invite new contributors and those who have not met the core development team.
 
-## How do I attend?
+## Schedule
 
-To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
-will open in early 2023.
+A preliminary schedule is as follows. The full JuliaCon runs from July 25-29. By
+purchasing a ticket to JuMP-dev, you can also attend the full JuliaCon
```

Comment:
The wording "purchasing a ticket to JuMP-dev" is a bit confusing given that below we talk about purchasing a ticket to JuliaCon. The tickets are for JuliaCon.

Reply:
I've changed the sentence.